pubsub/gcppubsub/gcppubsub.go

```diff
@@ -16,6 +16,13 @@
 // PubSub. Use OpenTopic to construct a *pubsub.Topic, and/or OpenSubscription
 // to construct a *pubsub.Subscription.
 //
+// Escaping
+//
+// Go CDK supports all UTF-8 strings; some strings are escaped (during writes)
```

Comment:
Ah I see. In that case, I think it would be more readable like this:

"""
Go CDK supports all UTF-8 strings; to make this work with providers lacking full UTF-8 support, strings must be escaped (during writes) and unescaped (during reads). The following escapes are required for gcppubsub:
"""

Reply:
Done.